scripts/alivecc.in

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+COMPILER=`basename "$0"`
+
+if [ $COMPILER = "alivecc" ]; then
+  EXE=clang
+elif [ $COMPILER = "alive++" ]; then
```

Comment:
sure we don't want to be symmetric/consistent and go with `alivec++` ?

Reply:
irrelevant to me. nuno can decide.

Reply:
(I'll just add that the analogy I was thinking of is with gcc/g++. s/g/alive/ and you get alivecc and alive++)

@nunoplopes: this script here is about as complex a program as I care to write in bash. if we want to start adding features such as detecting when the compiler is being invoked as a linker or assembler, I'll prefer to switch over to perl. since nobody else is likely to enjoy this, I am happy to commit to maintaining the code indefinitely. alternatively, we could use python or something else, in which case someone besides me should write the script.